File: Selenium/Activity5.py
from selenium import webdriver
from selenium.webdriver.common.by import By

with webdriver.Chrome() as driver:
		driver.get("https://training-support.net/webelements/dynamic-controls")
		print("Web age title - " + driver.title)
		checkBox = driver.find_element(By.ID, "checkbox")
		if(checkBox.is_displayed() == True):
			print("Checkbox is displayed")
		else:
			print("Checkbox is not displayed")
			
		toggle = driver.find_element(By.XPATH, "//button[text()='Toggle Checkbox']")
		toggle.click()
		if(checkBox.is_displayed() == True):
			print("Checkbox is displayed")
		else:
			print("Checkbox is not displayed")
		driver.quit()

File: Selenium/Activity5.java
package Selenium;

import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.WebElement;
import org.openqa.selenium.chrome.ChromeDriver;

public class Activity5 {

	public static void main(String[] args) {
		WebDriver driver = new ChromeDriver();
		driver.get("https://training-support.net/webelements/dynamic-controls");
		System.out.println("Page Title is : " + driver.getTitle());
		WebElement checkbox = driver.findElement(By.id("checkbox"));
		if(checkbox.isDisplayed() == true) {
			System.out.println("Checkbox is selected");
		}
		else {
			System.out.println("Checkbox is not selected");
		}
		driver.findElement(By.xpath("//button[text()='Toggle Checkbox']")).click();
		if(checkbox.isDisplayed() == true) {
			System.out.println("Checkbox is selected");
		}
		else {
			System.out.println("Checkbox is not selected");
		}
		driver.quit();
	}

}
